content/_index.md

```diff
@@ -34,6 +34,6 @@ The TCN Coalition serves as a gathering place to increase visibility across the
 
 ##  More Information
 
-More information on the TCN Protocol is available on GitHub at https://github.com/TCNCoalition/TCN
+More information on the TCN Protocol is available on GitHub at https://github.com/TCNCoalition/TCN.
```

Comment:
What do you think about making Github a hyperlink? E.g. 

```More information on the TCN Protocol is available on <a href="https://github.com/TCNCoalition/TCN">GitHub.</a>```

Reply:
Not sure if you're asking me, but if you are I think it's good for users to be able to see the actual URL, and I would recommend that the link open in a new tab.

Reply:
Okay then the URL would need to be enclosed in a target attribute.

Reply:
done.